include/boost/geometry/algorithms/detail/overlay/get_turns.hpp

```diff
@@ -741,7 +740,7 @@ struct get_turns_cs
         else if (c < left) return -1;
         else if (c > right) return 1;
         else return 0;
-    }
+    }*/
```

Comment:
You can remove this one. I don't remember exactly but I probably should have cleaned it up earlier.

Reply:
I removed the get_side function that was commented out and the commented out that was calling it. If the latter was meant to be kept, I can change the commit.

Reply:
This is completely fine, thanks!